.eslintignore

```diff
@@ -12,3 +12,5 @@ src/**/*.js
 src/**/*.map
 src/**/*.d.ts
 
+# Include eslint config
+!.eslintrc.js
```

Comment:
The above `*.js` rule (which makes this `!.eslintrc.js` necessary) is probably no longer needed -- `src/index.js` used to be at the root of the repository, which is why the `*.js` rule is still around.

I think we can just remove the above `*.js`, `*.map`, and `*.d.ts` rules and remove this line as well. The same applies to the `.gitignore` file as well.

Reply:
This actually requires more fixing; the underlying gitignore rules that I said could be removed are actually breaking the `examples/cameras/` project, and will be fixed in #19, so I'm not going to block this PR on this issue/fix.